docs/source/public/overview.rst

```diff
@@ -278,6 +278,50 @@ The directory should be removed and the job should be resubmitted (depending on
 
 2. The extract results script mentioned :ref:`here <inspecting results>` and the cleanup script mentioned :ref:`here <post-simulation cleanup>` both include output of which folder(s) and ``json`` file(s) contain errors and can be removed prior to starting new jobs.
 
+.. _fix_networks:
+
+Fixing broken networks
+----------------------
+
+If removing a reproducibly failing edge leads to a disconnected graph, new edges need to be added in order to fix the broken network.
```

Comment:
How would they know that this is a broken network? I.e. number of ligands lower than expected, failure during the analysis scripts....